CHANGELOG.md

```diff
@@ -2,6 +2,92 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 44 end 2016-08-1
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+44+Ending+Aug+1%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Simulation: RBAC filtering applied to Object Attributes
+  - Service Provisioning: Exposed number_of_vms when building the provision request for a service
+  - Service Dialogs: Support for “visible” flag for dynamic fields
+  - Expose Compliance and ComplianceDetail models
+  - New associations on VmOrTemplate and Host models:
+    - `expose :compliances`
+    - `expose :last_compliance`
+  - New Service Models
+    - Compliance: `expose :compliance_details`
+    - ComplianceDetail: `expose :compliance`, `expose :miq_policy`
+  - Generic Object: Service models created for GenericObject and GenericObjectDefinition
+- Platform
+  - PostgreSQL High Availability
+    - Added [repmgr](http://repmgr.org/)  to support automatic failover
```

Comment:
We added the package to the appliance in https://github.com/ManageIQ/manageiq-appliance-build/pull/142